crates/llm-base/src/model/mod.rs

```diff
@@ -207,6 +204,8 @@ pub struct ModelParameters {
     pub lora_adapters: Option<Vec<PathBuf>>,
     /// Whether to use GPU acceleration when available
     pub use_gpu: bool,
+    /// The number of layers to offload to the gpu. If `None`, all layers will be offloaded.
```

Comment:
I was confused at first - I thought that setting this to `None` would always offload all layers to the GPU - but then I saw the `use_gpu` above. I'd suggest altering the comment to mention that the layers will only be offloaded if `use_gpu` is set.